python-sdk/docs/CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 1.2.3
+
+### Bug fixes
+* Fix the dataframe operator behavior of appending to it's output table instead of replacing. When `if_exists` is set to `replace`, replace the table rather than append [#1260](https://github.com/astronomer/astro-sdk/issues/1260)
```

Comment:
```suggestion
* When `if_exists` is set to `replace` in dataframe operator, replace the table rather than append. This bug fix fixes a regression on the Dataframe operator which caused it to append content to an output table instead of replacing.  [#1260](https://github.com/astronomer/astro-sdk/issues/1260)
```

Reply:
[Done](https://github.com/astronomer/astro-sdk/pull/1289/commits/7599ef157ac85a6992512c9367fb4368ce9f4e86)